src/venn-client/index.ts

```diff
@@ -80,13 +80,14 @@ export class VennClient {
 
     /**
      * Approves a transaction request.
-     * @param {ethers.TransactionRequest} txData - The transaction request to approve. Must include to, from, value and data
+     * @param {ethers.TransactionRequest} txData - The transaction request to approve. Must include to, from, value and data. Chain ID is optional.
      * @param {string} txData.to - The recipient address of the transaction (required)
      * @param {string} txData.from - The sender address of the transaction (required)
      * @param {string} txData.value - The amount of Ether to send with the transaction (required)
      * @param {string} txData.data - The data payload of the transaction (required)
+     * @param {number} txData.chainId - Chain ID to which the transaction will be sent (optional). If omitted, the chain ID currently set in the user's wallet will be used. If unavailable, the transaction will fail
```

Comment:
I think that wrapping the `txData.chainId` in square brackets tells the IDE this is an optional parameter:

```suggestion
     * @param {number} [txData.chainId] - Chain ID to which the transaction will be sent (optional). If omitted, the chain ID currently set in the user's wallet will be used. If unavailable, the transaction will fail
```